apps/finance/app/public/images/icon.svg

```diff
@@ -1 +1,28 @@
-<svg width="22" height="22" viewBox="0 0 22 22" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="M0 0h22v22H0z"/><path d="M18.063 3c.253 0 .473.093.659.278a.901.901 0 0 1 .278.66v13.124a.901.901 0 0 1-.278.66.901.901 0 0 1-.66.278H4.938a.901.901 0 0 1-.659-.278.901.901 0 0 1-.278-.66V3.938c0-.253.093-.473.278-.659A.901.901 0 0 1 4.938 3h13.125zm0 14.063V3.938H4.938v13.124h13.125zM7.28 8.155a.463.463 0 0 1-.468-.469.45.45 0 0 1 .139-.33.45.45 0 0 1 .33-.138h.937V6.28a.45.45 0 0 1 .139-.33.45.45 0 0 1 .33-.138.45.45 0 0 1 .33.139.45.45 0 0 1 .138.33v.937h.938a.45.45 0 0 1 .33.139.45.45 0 0 1 .139.33.463.463 0 0 1-.469.469h-.938v.937a.463.463 0 0 1-.469.469.463.463 0 0 1-.468-.47v-.937H7.28zm3.062 3.501c.097.098.146.21.146.337a.417.417 0 0 1-.146.322l-.996.996.996.997c.097.097.146.21.146.337a.417.417 0 0 1-.146.322.425.425 0 0 1-.33.146.425.425 0 0 1-.33-.146l-.995-.982-.997.982a.425.425 0 0 1-.33.146.425.425 0 0 1-.329-.146.417.417 0 0 1-.146-.322c0-.127.049-.24.146-.337l.996-.996-.996-.997a.417.417 0 0 1-.146-.322c0-.127.049-.24.146-.337a.447.447 0 0 1 .33-.132c.132 0 .242.044.33.132l.995.996.997-.996a.447.447 0 0 1 .33-.132c.131 0 .24.044.329.132zm2.563-3.5a.463.463 0 0 1-.469-.47.45.45 0 0 1 .14-.33.45.45 0 0 1 .33-.138h2.812a.45.45 0 0 1 .33.139.45.45 0 0 1 .139.33.463.463 0 0 1-.469.469h-2.813zm0 4.687a.463.463 0 0 1-.469-.469.45.45 0 0 1 .14-.33.45.45 0 0 1 .33-.139h2.812a.45.45 0 0 1 .33.14.45.45 0 0 1 .139.329.463.463 0 0 1-.469.469h-2.813zm0 1.875a.463.463 0 0 1-.469-.469.45.45 0 0 1 .14-.33.45.45 0 0 1 .33-.139h2.812a.45.45 0 0 1 .33.14.45.45 0 0 1 .139.329.463.463 0 0 1-.469.469h-2.813z" fill="#8B9396"/></g></svg>
\ No newline at end of file
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
One thing: could we put all of these svgs through [SVGOMG](https://jakearchibald.github.io/svgomg/) so they can be optimized?